packages/client/katello-host-tools/katello-host-tools.spec

```diff
@@ -149,36 +149,54 @@ rm -rf %{buildroot}
 
 %if %{dnf_install}
 %global katello_libdir %{python3_sitelib}/katello
+%global python_libdir %{python3_sitelib}
 %global plugins_dir %{python3_sitelib}/dnf-plugins
 %global plugins_confdir %{_sysconfdir}/dnf/plugins
 %endif
 
 %if %{yum_install}
 %if 0%{?rhel} == 6
 %global katello_libdir %{python_sitelib}/katello
+%global python_libdir %{python_sitelib}
 %else
 %global katello_libdir %{python2_sitelib}/katello
+%global python_libdir %{python2_sitelib}
 %endif
 %global plugins_dir %{_usr}/lib/yum-plugins
 %global plugins_confdir %{_sysconfdir}/yum/pluginconf.d
 %endif
 
 %if %{zypper_install}
 %global katello_libdir %{python_sitelib}/katello
+%global python_libdir %{python_sitelib}
 %global plugins_dir %{_usr}/lib/zypp/plugins/commit/
 # Deploy yum plugin config to control reports
 %global plugins_confdir %{_sysconfdir}/yum/pluginconf.d
 %endif
 
+pushd src
+%if %{dnf_install} || 0%{?suse_version} >= 1500
+%py3_install
+%else
+%if 0%{?rhel} == 6
+%{__python} setup.py install
+%else
+%py_install
+%endif
+%endif
+popd
+
 mkdir -p %{buildroot}%{katello_libdir}
 mkdir -p %{buildroot}%{plugins_dir}
+mkdir -p %{buildroot}%{_sbindir}
+mv %{buildroot}%{_bindir}/* %{buildroot}%{_sbindir}/
 
-cp src/katello/*.py %{buildroot}%{katello_libdir}/
+# cp src/katello/*.py %{buildroot}%{katello_libdir}/
```

Comment:
Can we drop the commented cod if it's no longer needed?

Reply:
will do